src/simpleGit.ts

```diff
@@ -20,8 +20,10 @@ export class SimpleGit extends GitManager {
         if (this.isGitInstalled()) {
             const adapter = this.app.vault.adapter as FileSystemAdapter;
             const path = adapter.getBasePath();
+            const extraPath = this.plugin.settings.basePath || ""
+            console.log(path + extraPath)
             this.git = simpleGit({
-                baseDir: path,
+                baseDir: path + extraPath,
```

Comment:
`extraPath` has to start with an `/`, doesn't it? Did you specify `extraPath` with a leading `/` in your testing? That's weird, because it's a relative path.

Reply:
Great feedback. I updated the plugin to use relative paths.

Reply:
I added a commit myself using platform specific separator, because Windows uses `\`.